### DISCUSSION TITLE
Error messages at leaving MusE

### DISCUSSION THREAD

**Discussion by kybos** [muse-sequencer/muse#1154] [Team Posts]
Since some time, I get this three lines at the end of the console output:

  Error: plugin is not NULL
  Error: plugin is not NULL
  Error: plugin is not NULL

It seems to come from the new audio converter.
The errors are there even if I execute MusE with the -h switch (muse3 -h) to only display the help.

**Comment by terminator356** [comment]
Thanks for reminding me.
I think that's a long-time weirdness with our plugin list after running MusE...

Wait... what? It happens with -h ???
Whoa, now that's a new one ! That shouldn't happen. I'll take a look.

**Comment by terminator356** [comment]
OK This should be fixed now. Thanks.
The error was benign. The next line closes the library and releases all memory.

But I did discover that I did not delete MusEGlobal::defaultAudioConverterSettings
 inside of the auto-restart loop in main. So I moved in there.
And I neglected to clear/delete audioConverterPluginList at all.
This is fixed now, done inside of the auto-restart loop in main.
Tested OK auto-restarting MusE several times with stretch converters employed.

BTW: About auto-restart:
I has occurred to me that I really ought to re-load the song that the user was working on !
Currently it just restarts with the default song.
